### DISCUSSION TITLE
Automate Clock Volume Network Power System Icons

### DISCUSSION THREAD

**Discussion by schmerold** [valinet/ExplorerPatcher#343] [Q&A]
Does anyone have a solution to automatically enable system icons?
I have had some success with the clock using Nirsoft: 
nircmd.exe win child class "Shell_TrayWnd" show class "TrayClockWClass"

With nircmd, the clock appears for a few minutes, then is disabled shortly thereafter. 

This allows the end-user to easily make the change, too many end users throw their hands up and say "It's too complicate"
"explorer.exe shell:::{05d7b0f4-2121-4eff-bf6b-ed3f69b894d9} \SystemIcons"

Please help!

**Comment by valinet** [comment]
Implemented natively in the latest version. Thanks.

**Reply by schmerold** [reply]
That's great news - please refer me to the documentation for this feature - I thought I read it cover to cover, but seem to have missed this implementation.

**Reply by sisericm** [reply]
@schmerold, check the release notes for the latest release. 
[https://github.com/valinet/ExplorerPatcher/releases/tag/22000.318.37.2_abf0d38](url)

> Implemented a proper right click menu for the Windows 11 taskbar - it displays the most common options, similar to previous Windows releases and the Windows 10 taskbar, including frequently accessed items like "Taskbar settings", "Task Manager" and "Show the desktop"
System tray icons are now left intact when switching between the Windows 10 and Windows 11 taskbars, or when switching builds, reinstalling the application etc. Basically, now, once you set a certain layout for the system tray with the Windows 10 taskbar, it will always be remembered, instead of the annoying behavior where Windows was discarding your choices in order to accommodate the Windows 11 taskbar

**Reply by valinet** [reply]
Thanks @sisericm. @schmerold the "documentation" changes from time to time and also is a bit lacking at the moment, to put it mildly, but I will try to provide better docs now that the app has most of the features needed, I can concentrate on improving these other aspects as well. it was mainly an issue of time, or lack of. Thanks.
